File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/JAVA/Pattern.java
/*
Program 5 : Write a Program to print following Pattern.
Output :
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
*/

class Pattern{
	
	public static void main(String args[]){
		
		for(int outeritr = 1 ; outeritr <= 4 ; outeritr++ ){

			for(int inneritr = 1 ; inneritr <= 4 ; inneritr++ ){

				System.out.printf("0 ");
			}
			System.out.printf("\n");
		}
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/C/pattern.c
/*
Program 5 : Write a Program to print following Pattern.
Output :
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
*/

#include<stdio.h>

#define lowerlimit 1
#define upperlimit 4

void main(){
	
	for(int outeritr = lowerlimit ; outeritr <= upperlimit ; outeritr++ ){

		for(int inneritr = lowerlimit ; inneritr <= upperlimit ; inneritr++ ){
			
			printf("0 ");
		}
		printf("\n");	
	}	
}	
